tests/unit_tests/test_stats.py

```diff
@@ -65,8 +79,14 @@ def test_uniform():
     assert t.p == [1/(b-a), 1/(b-a)]
     assert t.interpolation == 'histogram'
 
+    exp_mean = 0.5 * (a + b)
+    n_samples = 1_000_000
+    samples = d.sample(n_samples, seed=100)
+    assert samples.mean() == pytest.approx(exp_mean, rel=1e-02)
```

Comment:
Probably would be better to check that mean is within some number of standard deviations rather than hard-coding the relative error tolerance